.changeset/rich-oranges-attack.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": minor
+---
+
+fix: Use queue name, not ID, for event notification subcommands
+
+This is a breaking change over previous behavior.
```

Comment:
Sorry to be a pain, this will confuse users because they will see a breaking change in a minor release. Proposed change:

```suggestion
Since the original command was not yet operational, this update does not constitute a breaking change.
```

Reply:
Not a pain, good call. I wasn't sure the right thing to do here.